### DISCUSSION TITLE
recipes/breads/vermont-sourdough/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1166] [Comments]
# recipes/breads/vermont-sourdough/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/vermont-sourdough/

**Comment by nicholaswilde** [comment]
- 210g Sharp cheddar cheese, shredded or cubed
- 67g Jalapeño, finely diced (1–2 large peppers)

**Reply by nicholaswilde** [reply]
- Use bread flour if using jalapenos and cheddar.

**Reply by nicholaswilde** [reply]
- 34g Honey

**Reply by nicholaswilde** [reply]
- 23.00% Green olives, rinsed and chopped, if desired (any green olive variety will work well)
- 0.75% Herbes de Provence
